variables.tf

```diff
@@ -1,3 +1,15 @@
+variable "config_aggregator_name" {
+  description = "The name of the aggregator."
+  type        = "string"
+  default     = ""
```

Comment:
Do we want to just add a default here? Something like "organization"? If you only have one most of the time anyway...

Reply:
done